### DISCUSSION TITLE
Ad revenue

### DISCUSSION THREAD

**Discussion by Ducci9129** [EmulatorJS/EmulatorJS#479] [Q&A]
Just a quick question, would you still get the ad revenue if it's hosted on another website. (mine for example) I feel kinda guilty using your work but seeing that you have ads disabled except on your web domain.

**Comment by ethanaobrien** [comment]
No, I only have ads on the main demo site. Don't worry about ad revenue, it's completely ok. I made it to where it would only show on the demo page specifically for that. Don't worry about it, you're fine
